virttest/ssh_key.py

```diff
@@ -32,31 +32,31 @@ def get_public_key():
     has_rsa_keypair = (os.path.isfile(rsa_public_key_path) and
                        os.path.isfile(rsa_private_key_path))
 
-    if has_dsa_keypair:
-        print('DSA keypair found, using it')
-        public_key_path = dsa_public_key_path
-
-    elif has_rsa_keypair:
-        print('RSA keypair found, using it')
+    if has_rsa_keypair:
+        logging.info('RSA keypair found, using it')
         public_key_path = rsa_public_key_path
 
-    else:
-        print('Neither RSA nor DSA keypair found, creating DSA ssh key pair')
-        process.system('ssh-keygen -t dsa -q -N "" -f %s' %
-                       dsa_private_key_path)
+    elif has_dsa_keypair:
+        logging.info('DSA keypair found, using it')
         public_key_path = dsa_public_key_path
 
+    else:
+        logging.info('Neither RSA nor DSA keypair found, creating DSA ssh key pair')
```

Comment:
should be "creating RSA  ssh key pair ", right?

Reply:
Fixed, thx